docs/notebooks/Distributed_arrays_and_automatic_parallelization.md

```diff
@@ -13,7 +13,7 @@ kernelspec:
 
 +++ {"id": "PxHrg4Cjuapm"}
 
-# Parallelism with JAX
+# Distributed arrays and automatic parallelization
```

Comment:
Rename the file according to the title too?

Reply:
Ah already done locally, sorry please hold off on review until I push again.